### PR TITLE
docs: Chinese documentation for loki-logger

### PR DESCRIPTION
**Pull request by mfordjody**
### Description

Fixes https://github.com/apache/apisix/issues/9676 (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/loki-logger.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+| keepalive     | boolean       | False    | true | 当设置为 `true` 时，为多个请求保持连接。 |
+| keepalive_timeout | integer       | False    | 60000ms | 空闲时间之后连接被关闭。大于或等于 1000ms 的范围。  |
+| keepalive_pool | integer       | False    | 5       | 连接池限制。大于或等于 1 的范围。 |
+| log_format | object | False    |          | 日志格式以 JSON 格式的键值对声明。值只支持字符串。[APISIX 变量](../apisix-variable.md) 和 [Nginx 变量](http://nginx.org/en/docs/varindex.html) 可以通过在字符串前加上 `$` 来使用。 |
+| include_req_body   | boolean | False    | false | 当设置为 `true` 时，在日志中包括请求正文。如果请求正文太大，不能保存在内存中，由于 Nginx 的限制，它不能被记录下来。 |
+| include_req_body_expr | array   | False    |  | 当 `include_req_body` 属性被设置为 `true` 时的过滤器。只有当这里设置的表达式评估为 `true` 时，才会记录请求正文。更多信息请参见 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 |
+| include_resp_body  | boolean | False    | false | 当设置为 `true` 时，在日志中包括响应体。 |
+| include_resp_body_expr | array   | False    |  | 当 `include_resp_body` 属性被设置为 `true` 时的过滤器。只有当这里设置的表达式评估为 `true` 时，才会记录响应体。更多信息请参见 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 |
+
+这个插件支持使用批处理程序来聚集和处理条目（日志/数据）的批次。这避免了频繁提交数据的需要。批处理程序每 `5` 秒或当队列中的数据达到 `1000` 时提交数据。参见 [批处理程序](../batch-processor.md#configuration) 了解更多信息或设置你的自定义配置。
```
Comment (lingsamuel) [inline docs/zh/latest/plugins/loki-logger.md:56]: 我们使用 `批处理器` 这个名词（参考：https://github.com/apache/apisix/blob/master/docs/zh/latest/batch-processor.md）

File: docs/zh/latest/plugins/loki-logger.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+## 元数据
+
+你还可以通过配置插件元数据来设置日志的格式。以下配置是可用的：
+
+| 名称 | 类型 | 必选项 | 默认值 | 描述 |
+|------|------|----------|--|-------------|
+| log_format | object | False | {"host": "$host", "@timestamp": "$time_iso8601", "client_ip": "$remote_addr"} | Log format declared as key value pairs in JSON format. Values only support strings. [APISIX variables](../apisix-variable.md) and [Nginx variables](http://nginx.org/en/docs/varindex.html) can be used by prefixing the string with `$`. |
```
Comment (lingsamuel) [inline docs/zh/latest/plugins/loki-logger.md:64]: 这里没有翻译

File: docs/zh/latest/plugins/loki-logger.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+### 日志不能正常推送
+
+看一下 `error.log` 这样的日志。
+
+```text
+2023/04/30 13:45:46 [error] 19381#19381: *1075673 [lua] batch-processor.lua:95: Batch Processor[loki logger] failed to process entries: loki server returned status: 401, body: no org id, context: ngx.timer, client: 127.0.0.1, server: 0.0.0.0:9081
+```
+
+该错误可以根据 `未能处理条目：loki 服务器返回状态： 401, body: no org id` 和 loki 服务器的响应体。
```
Comment (lingsamuel) [inline docs/zh/latest/plugins/loki-logger.md:151]: The error message shouldn't be translated here

File: docs/zh/latest/plugins/loki-logger.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+### 当 RPS 较高时出现错误？
+
+- 确保 `keepalive` 相关的配置被正确设置。更多信息见 [Attributes](#attributes)。
```
Comment (lingsamuel) [inline docs/zh/latest/plugins/loki-logger.md:155]: The anchor link is incorrect

File: docs/zh/latest/plugins/loki-logger.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+- 检查 `error.log` 中的日志，寻找这样的日志记录。
+
+    ```text
+    2023/04/30 13:49:34 [error] 19381#19381: *1082680 [lua] batch-processor.lua:95: Batch Processor[loki logger] failed to process entries: loki server returned status: 429, body: Ingestion rate limit exceeded for user tenant_1 (limit: 4194304 bytes/sec) while attempting to ingest '1000' lines totaling '616307' bytes, reduce log volume or contact your Loki administrator to see if the limit can be increased, context: ngx.timer, client: 127.0.0.1, server: 0.0.0.0:9081
+    ```
+
+  - 通常与高 QPS 相关的日志看起来像上面的内容。错误是：`用户 tenant_1（限制：4194304 字节/秒）在试图摄取 '1000' 行共计 '616307' 字节时超过了消化率限制，减少日志量或联系你的 Loki 管理员，看看是否可以增加限制`。
```
Comment (lingsamuel) [inline docs/zh/latest/plugins/loki-logger.md:162]: Same as above. The error message should be kept as is

File: docs/zh/latest/plugins/loki-logger.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+| keepalive     | boolean       | False    | true | 当设置为 `true` 时，为多个请求保持连接。 |
+| keepalive_timeout | integer       | False    | 60000ms | 空闲时间之后连接被关闭。大于或等于 1000ms 的范围。  |
+| keepalive_pool | integer       | False    | 5       | 连接池限制。大于或等于 1 的范围。 |
+| log_format | object | False    |          | 日志格式以 JSON 格式的键值对声明。值只支持字符串。[APISIX 变量](../apisix-variable.md) 和 [Nginx 变量](http://nginx.org/en/docs/varindex.html) 可以通过在字符串前加上 `$` 来使用。 |
+| include_req_body   | boolean | False    | false | 当设置为 `true` 时，在日志中包括请求正文。如果请求正文太大，不能保存在内存中，由于 Nginx 的限制，它不能被记录下来。 |
+| include_req_body_expr | array   | False    |  | 当 `include_req_body` 属性被设置为 `true` 时的过滤器。只有当这里设置的表达式评估为 `true` 时，才会记录请求正文。更多信息请参见 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 |
+| include_resp_body  | boolean | False    | false | 当设置为 `true` 时，在日志中包括响应体。 |
+| include_resp_body_expr | array   | False    |  | 当 `include_resp_body` 属性被设置为 `true` 时的过滤器。只有当这里设置的表达式评估为 `true` 时，才会记录响应体。更多信息请参见 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 |
+
+这个插件支持使用批处理器来聚集和处理条目（日志/数据）的批次。这避免了频繁提交数据的需要。批处理程序每 `5` 秒或当队列中的数据达到 `1000` 时提交数据。参见 [批处理程序](../batch-processor.md#configuration) 了解更多信息或设置你的自定义配置。
```
Comment (lingsamuel) [inline docs/zh/latest/plugins/loki-logger.md:56]: Last `批处理程序` need to be changed

**Review by lingsamuel — APPROVED** [review]
LGTM, just a small change is needed

**Review by Neilblaze — APPROVED** [review]
SGTM!

File: docs/zh/latest/plugins/loki-logger.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+当插件启用时，APISIX 将把请求上下文信息序列化为 [JSON 中的日志条目](https://grafana.com/docs/loki/latest/api/#push-log-entries-to-loki) 并提交给批处理队列。当超过最大批处理量时，队列中的数据会被推送到 Grafana Loki。更多细节见 [batch processor](../batch-processor.md)。
+
+## 属性
+
+| 名称 | 类型 | 必选项 | 默认值 | 描述 |
+|--|---|---|---|---|
+| endpoint_addrs | array[string] | True |  | Loki API 基础 URL，格式如 http://127.0.0.1:3100，支持 HTTPS 和域名。如果配置了多个端点，它们将被随机写入。 |
+| endpoint_uri | string | False | /loki/api/v1/push | 如果你使用与 Loki Push API 兼容的日志收集服务，你可以使用这个配置项来定制 API 路径。 |
+| tenant_id | string | False | fake | Loki 租户 ID。根据 Loki 的 [多租户文档](https://grafana.com/docs/loki/latest/operations/multi-tenancy/#multi-tenancy)，其默认值被设置为单租户下的默认值 `fake`。 |
+| log_labels | object | False | {job = "apisix"} | Loki 日志标签。[APISIX 变量](../apisix-variable.md) 和 [Nginx 变量](http://nginx.org/en/docs/varindex.html) 可以通过在字符串前加上 `$` 来使用，包括单独和组合，如 `$host` 或 `$remote_addr:$remote_port`。 |
```
Comment (monkeyDluffy6017) [inline docs/zh/latest/plugins/loki-logger.md:44]:
```suggestion
| log_labels | object | False | {job = "apisix"} | Loki 日志标签。[APISIX 变量](../apisix-variable.md) 和 [Nginx 变量](http://nginx.org/en/docs/varindex.html) 可以通过在字符串前加上 `$` 来使用，变量可以单独或者组合使用，如 `$host` 或 `$remote_addr:$remote_port`。 |
```

File: docs/zh/latest/plugins/loki-logger.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+| ssl_verify    | boolean       | False    | true | 当设置为 `true` 时，验证 SSL 证书。|
+| timeout       | integer       | False    | 3000ms | Loki 服务 HTTP 调用的超时。范围从 1 到 60,000ms。  |
```
Comment (monkeyDluffy6017) [inline docs/zh/latest/plugins/loki-logger.md:46]:
```suggestion
| timeout       | integer       | False    | 3000ms | Loki 服务 HTTP 请求的超时时间。范围从 1 到 60,000ms。  |
```

File: docs/zh/latest/plugins/loki-logger.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+| keepalive     | boolean       | False    | true | 当设置为 `true` 时，为多个请求保持连接。 |
+| keepalive_timeout | integer       | False    | 60000ms | 空闲时间之后连接被关闭。大于或等于 1000ms 的范围。  |
+| keepalive_pool | integer       | False    | 5       | 连接池限制。大于或等于 1 的范围。 |
+| log_format | object | False    |          | 日志格式以 JSON 格式的键值对声明。值只支持字符串。[APISIX 变量](../apisix-variable.md) 和 [Nginx 变量](http://nginx.org/en/docs/varindex.html) 可以通过在字符串前加上 `$` 来使用。 |
+| include_req_body   | boolean | False    | false | 当设置为 `true` 时，在日志中包括请求正文。如果请求正文太大，不能保存在内存中，由于 Nginx 的限制，它不能被记录下来。 |
+| include_req_body_expr | array   | False    |  | 当 `include_req_body` 属性被设置为 `true` 时的过滤器。只有当这里设置的表达式评估为 `true` 时，才会记录请求正文。更多信息请参见 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 |
+| include_resp_body  | boolean | False    | false | 当设置为 `true` 时，在日志中包括响应体。 |
+| include_resp_body_expr | array   | False    |  | 当 `include_resp_body` 属性被设置为 `true` 时的过滤器。只有当这里设置的表达式评估为 `true` 时，才会记录响应体。更多信息请参见 [lua-resty-expr](https://github.com/api7/lua-resty-expr)。 |
```
Comment (monkeyDluffy6017) [inline docs/zh/latest/plugins/loki-logger.md:54]: Please read it yourself first, don't rely on translation software

Comment (Neilblaze) [reply docs/zh/latest/plugins/loki-logger.md:54]: @monkeyDluffy6017 Would you mind recommending one for `ZH` to `EN`? I generally use [DeepL](https://www.deepl.com/en/translator)

Comment (monkeyDluffy6017) [reply docs/zh/latest/plugins/loki-logger.md:54]: I use DeepL and chatgpt, but i found that chatgpt is better

**Review by lingsamuel — APPROVED** [review]

**Comment by moonming** [comment]
@monkeyDluffy6017 ping

**Comment by monkeyDluffy6017** [comment]
@moonming I have checked this, I don't think the translation is of high quality, maybe it's just from the translation software

**Comment by monkeyDluffy6017** [comment]
@mfordjody Try to use the chatgpt to translate and make little change

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by monkeyDluffy6017 — CHANGES_REQUESTED** [review]
Please fix the lint error

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by leslie-tsang — APPROVED** [review]

**Pull request merged**